sys/vm/vm_page.c

```diff
@@ -2200,7 +2200,7 @@ vm_page_alloc_contig(vm_object_t object, vm_pindex_t pindex, int req,
 	return (m);
 }
 
-#ifdef CHERI_PURECAP_KERNEL
+#ifdef __CHERI_PURE_CAPABILITY__
 /*
```

Comment:
I'm curious if we still need the hack?  The compiler doesn't reorder the `csetbounds` anymore IIRC.

Reply:
Yeah we should GC this. Performance and ugliness aside, it's only a matter of time until someone touches the body of the loop upstream and we risk having this get out of sync.

Reply:
We don't, I have a TODO item in my list to revert it.